.gitignore

```diff
@@ -8,6 +8,10 @@ _obj
 _test
 tmp
 
+# IDE config files
+.idea
+.vscode
+
```

Comment:
```suggestion
```

Reply:
Unrelated

Reply:
I have been guilty of this. Turns out there is a global `.gitignore` mechanism. Works like a charm, no need to set this kind of thing on a per project basis.

* https://gist.github.com/subfuzion/db7f57fff2fb6998a16c
* https://git-scm.com/docs/gitignore

Reply:
@madkins23 That's a neat thing to know! Thanks for sharing.